guides/06_client-libraries/fastapi-app-with-the-gradio-client.md

```diff
@@ -0,0 +1,261 @@
+# Building a FastAPI App with the Gradio Python Client
+
+Tags: CLIENT, API, WEB APP
+
+In this blog post, we will demonstrate how to use the `gradio_client` Python library, which enables developers to make requests to a Gradio app programmatically, within a FastAPI web app. The web app we will be building is called "Acapellify," and it will allow users to upload video files as input and return a version of that video without instrumental music.
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/gradio-guides/acapellify.png)
+
+
+**Prequisites**
+
+Before we begin, make sure you are running Python 3.9 or later, and have the following libraries installed:
+
+* `gradio_client`
+* `fastapi`
+* `ffmpeg`
+
+You can install these libraries from `pip`: 
+
+``` bash
+$ pip install gradio_client fastapi ffmpeg
```

Comment:
is this actually correct for ffmpeg? or is it ffmpeg-python?

Reply:
Yikes good catch